apps/web/app/(all)/[workspaceSlug]/(settings)/settings/account/security/page.tsx

```diff
@@ -87,8 +87,10 @@ const SecurityPage = observer(() => {
         title: t("auth.common.password.toast.change_password.success.title"),
         message: t("auth.common.password.toast.change_password.success.message"),
       });
-    } catch (err: any) {
-      const errorInfo = authErrorHandler(err.error_code?.toString());
+    } catch (error: unknown) {
+      const err = error as Error & { error_code?: string };
```

Comment:
You should check if it's an object with the key too anything can be thrown in a catch statement

Reply:
ah right! one thing i don't like about typescript - it's causing me to stop thinking about what's really happening